state-chain/runtime/src/migrations/increase_arb_safety_margin.rs

```diff
@@ -0,0 +1,40 @@
+use crate::Runtime;
+use cf_chains::instances::ArbitrumInstance;
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+use pallet_cf_ingress_egress::WitnessSafetyMargin;
+#[cfg(feature = "try-runtime")]
+use sp_runtime::DispatchError;
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
+
+use codec::{Decode, Encode};
+
+pub struct Migration;
+
+// Keeping it a multiple of 24 to match the witness period of Arbitrum
+const NEW_ARB_SAFETY_MARGIN: u64 = 672;
```

Comment:
Is this correct? The current value is 1...

cc @kylezs

Reply:
Well, this is what we would have if we were also to increase ETH to 2+ mins - this is the value I had here: https://github.com/chainflip-io/chainflip-backend/pull/5686

We can probably lower it to closer to what ETH is though I think: 6 (eth margin) * 14 (eth block time in secs) * 4(arb blocks per second) = 336

Reply:
Ok but I still don't understand why it's currently 1, particularly in the context of the comment that says we're "keeping" it a multiple of 24.